File: src/main/java/nl/vollo/kern/repository/GebruikerRepository.java
package nl.vollo.kern.repository;

import nl.vollo.kern.model.Gebruiker;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface GebruikerRepository extends JpaRepository<Gebruiker, Long> {
    Optional<Gebruiker> findByGebruikersnaam(String gebruikersnaam);
}

File: src/main/kotlin/nl/vollo/kern/repository/GebruikerRepository.kt
package nl.vollo.kern.repository

import nl.vollo.kern.model.Gebruiker
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface GebruikerRepository : JpaRepository<Gebruiker, Long> {
    fun findByGebruikersnaam(gebruikersnaam: String): Optional<Gebruiker>
}
